_common/schemas/__init__.py

```diff
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .base import APIResponse, Base, OutputMode
+from .base import APIDictResponse, APIResponse, Base, OutputMode
```

Comment:
I would try to avoid relative imports. You can see here [why](https://realpython.com/absolute-vs-relative-python-imports/#pros-and-cons-of-absolute-imports)